src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -699,9 +679,22 @@ internal override LocalSymbol GetAdjustedLocalSymbol(LocalSymbol local, int posi
             return adjustedLocal;
         }
 
-        private LocalFunctionSymbol GetDeclaredLocalFunction(LocalFunctionStatementSyntax declarationSyntax, SyntaxToken declaredIdentifier)
+        private LocalFunctionSymbol GetDeclaredLocalFunction(LocalFunctionStatementSyntax declarationSyntax)
```

Comment:
It feels like we would want to remap type parameters of local functions as well.